spec/board_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require './lib/tic_tac_toe.rb'
+
+RSpec.describe Board do
```

Comment:
#### `REQUIRED`

Consider using `let` instead, this file have 6 different place `board = Board.new` was defined. Using the below suggestion will avoid defining `board` in each example. Learn more [here](https://github.com/rubocop-hq/rspec-style-guide#let-blocks)

```suggestion
RSpec.describe Board do
  let(:board) { Board.new }
  
  describe ' #initialize_Board ' do
```